src/jaxsim/api/model.py

```diff
@@ -396,9 +396,7 @@ def total_mass(model: JaxSimModel) -> jtp.Float:
     """
 
     return (
-        jax.vmap(lambda idx: js.link.mass(model=model, link_index=idx))(
-            jnp.arange(model.number_of_links())
-        )
+        js.link.mass(model=model, link_index=jnp.arange(model.number_of_links()))
```

Comment:
Thanks Diego! Addressed in https://github.com/ami-iit/jaxsim/pull/235/commits/bd52dd2b28eb13b8310b0530bfedfb66b82c7361

The exception was still working as by passing an array instead of a float in `link_index`, analyzing:
```python
condition=jnp.array(
            [link_index < 0, link_index >= model.number_of_links()]
        ).any()
```
from `jnp.array( < 0, link_index >= model.number_of_links()])` you get an array of booleans of shape `(model.number_of_links(),)`, and by adding `.any()` you still get a boolean that can be used in the `jax.lax.cond` in `jaxsim.exceptions`.